inspirehep/records/api/base.py

```diff
@@ -1098,3 +1102,18 @@ def get_earliest_date(self):
             date = earliest_date(dates, full_date=True)
             return date
         return None
+
+    def get_citations_count(self):
+        if self.pid_type in ["lit", "dat"]:
+            return len(self.model.citations)
+        raise CitationsNotSupportedForThisPidType
+
+    @property
+    def citation_count(self):
+        return self.get_citations_count()
```

Comment:
Method was requested in task, and property is for serializers (so we won't have to call any function - `citation_count = fields.Raw(dump_only=True)`)

Reply:
on serializers we can use ``fields.Method``, we don't need to add a ``property``

Reply:
Is there any benefit of not using property in here?

Reply:
I'm not convinced that we need a property for citation count, in that case we should have for ``earliest_date`` etc

Reply:
I think is not a bad design. Actually, I would rather keep only the property and remove the function at all.
Indeed, we should agree on a style and keep it consistent, but since there was no decision ahead, every option is still feasible.

Reply:
It's not a bad design, it's just that these values are calculated and not part of the record metadata, if we go for property we have to do it for ``earliest_date`` as well

Reply:
done